scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -43,50 +46,76 @@ workflow ImportGenomes {
     }
   }
 
-  call LoadData as LoadMetadataTsvs{
-    input:
-      done = CreateImportTsvs.done[0],
-      project_id = project_id,
-      dataset_name = dataset_name,
-      storage_location = output_directory,
-      datatype = "metadata",
-      numbered = "false",
-      partitioned = "false",
-      table_id = table_id,
-      schema = metadata_schema,
-      preemptible_tries = preemptible_tries,
-      docker = docker_final
-  }
+  scatter (i in range(GetMaxTableId.max_table_id)) {
+    call LoadData as LoadMetadataTsvs{
+      input:
+        done = CreateImportTsvs.done[0],
+        project_id = project_id,
+        dataset_name = dataset_name,
+        storage_location = output_directory,
+        datatype = "metadata",
+        numbered = "false",
+        partitioned = "false",
+        table_id = i + 1,
+        schema = metadata_schema,
+        preemptible_tries = preemptible_tries,
+        docker = docker_final
+    }
 
     call LoadData as LoadPetTsvs{
-    input:
-      done = CreateImportTsvs.done[0],
-      project_id = project_id,
-      dataset_name = dataset_name,
-      storage_location = output_directory,
-      datatype = "pet",
-      table_id = table_id,
-      schema = pet_schema,
-      preemptible_tries = preemptible_tries,
-      docker = docker_final
+      input:
+        done = CreateImportTsvs.done[0],
```

Comment:
Does Cromwell wait until the full scatter of `CreateImportTsvs` is complete? Or will this run as soon as the first shard is done?

Reply:
I didn't change this -- so I assumed it was doing the right thing before (it does seem to wait) but I wonder if instead if "done" should be an array of all the inputs?   @ahaessly how did you arrive at this solution?

Reply:
i think that when i used done, it was just a true/false that was set at the end of a task. i don't think i used it after a scatter. but i think passing the array would be safest. i guess we would change the input param to be an Array of String? (if it's an array of file, we should set the meta param for localizationOptional to true so it doesn't try to localize them all)

Reply:
Yeah I agree, we should pass in the whole Array.